sprint0/sprint0/Weapons/ItemsWeaponsSpriteFactory.cs

```diff
@@ -11,12 +11,14 @@ namespace sprint0
     public class ItemsWeaponsSpriteFactory
     {
         private readonly Game1 game;
-        private readonly Texture2D texture;
+        private readonly Texture2D texture1;
+        private readonly Texture2D texture2;
 
         public ItemsWeaponsSpriteFactory(Game1 game)
         {
             this.game = game;
-            texture = game.Content.Load<Texture2D>("Images/ItemsAndWeapons");
+            texture1 = game.Content.Load<Texture2D>("Images/ItemsAndWeapons");
+            texture2 = game.Content.Load<Texture2D>("Images/Link");
```

Comment:
maybe rename these to reflect items/weapons and link but maybe that also doesn't make sense